src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -797,6 +797,8 @@ unique_ptr<Scene> svgSceneBuild(SvgNode* node, float& vx, float& vy, float& vw,
     Box vBox = {vx, vy, vw, vh};
     auto docNode = _sceneBuildHelper(node, vBox, svgPath, false, 0);
     _applySvgViewFlag(docNode.get(), vx, vy, vw, vh, w, h, viewFlag);
+    w *= sw;
+    h *= sh;
```

Comment:
Header() is designed to return necessary information to the user. However, since header() does not account for percentages, users might acquire incorrect results. Unless we can understand each other, I may have to give you a negative rating.

Reply:
in a case no viewbox was given and:
- now width/height 
or
- widrth and height  were given in %
header doesn;t have info about the actual width and height. I see that you've made some changes here https://github.com/thorvg/thorvg/commit/842dd383e2c755541ab7adb2a8942a1dbddf64f2 so probably there you are solving this (didn;t check very carefully yet), but at this stage I can not change this. 
do you want me to wait with this pr until your changes are applied?

Reply:
@mgrudzinska  Okay, I understand your point. Since you've established the sizing policy for SVG, which includes factors such as pt, %, viewport, and viewbox, I would like to request that you include our current implementation policy documentation within the code as comments. The logic is a bit complex, and the rules are scattered and incomplete, so we need to establish concrete rules to prevent them from being broken by someone else in the future. Any comments and opinions are welcome.

As an example, we could include the following comments within the code (in header function or a new unified sizing method) to outline the SVG sizing policy:

For example:

/*  Here is the SVG sizing policy. Derive the SVG size based on the next priority: 
 1. Use ViewSize if it's valid: 
    - if viewSize is in Percentage, then calc the size after applying it to the Viewbox size.
 3. ViewSize is invalid, Use Viewbox.
    - ...
4. Invalid ViewSize & Viewbox? 
  - GetBound from the actual content and use the size as ...
5. ...  */